assets/sass/v2/_consent-email.scss

```diff
@@ -15,11 +15,41 @@
   }
 }
 
+.enduser-email-otp-only {
+  .o-form-button-bar {
+    display: flex;
+  }
+  &--info {
+    display: flex;
+    margin-top: 0.5rem;
+    margin-bottom: 0.5rem;
+    align-items: center;
+  }
+  &--icon {
+    margin-right: 1rem;
+    height: 2rem;
+    width: 2rem;
+    background-size: cover;
+  }
+}
+
+.enter-code-on-page {
+  font-size: small;
+  margin-bottom: 1.5rem;
+  text-align: center;
+}
+
 .otp-value {
```

Comment:
nit: I think you can remove this since it was only used here, or use this as the name for your class instead of `.otp-value-with-margin-bottom`, `.otp-value` is a simpler name

Reply:
Good point, will keep the name `.otp-value`.